services/app-api/src/emailer/templateHelpers.ts

```diff
@@ -181,6 +182,68 @@ const generateCMSReviewerEmailsForUnlockedContract = (
     return pruneDuplicateEmails(reviewers)
 }
 
+/* 
+    Determine reviewers for a given health plan package and state
+    - devReviewTeamEmails added to all emails by default
+    - dmcpSubmissionEmails added in both CONTRACT_ONLY and CONTRACT_AND_RATES
+    - oactEmails added for CONTRACT_AND_RATES
+    - dmco is added to emails via state analysts
+    
+    Return should be wrapped in pruneDuplicate to ensure even if config is added twice, we get unique list of reviewers
+*/
+const generateCMSReviewerEmailsForSubmittedContract = (
```

Comment:
nit: Lot's of duplicate code with `generateCMSReviewerEmailsForUnlockedContract`. I think the only difference is getting data from the `draftRevision` vs `packageSubmissions`, we could probably do something to consolidate the functions together.